src/coreclr/src/System.Private.CoreLib/src/System/ValueType.cs

```diff
@@ -20,6 +21,8 @@ namespace System
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public abstract class ValueType
     {
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2006:UnrecognizedReflectionPattern",
+            Justification = "Unused fields don't make a difference for equality")]
```

Comment:
What do you think of adding "trimmed" to this message?

"Unused fields that are trimmed don't make a difference for equality"